### PR TITLE
Pl 31585

### PR DESCRIPTION
**Pull request by rijajoo**
## Describe your changes

Fixed User Resource

## Comment Triggers

<details>
  <summary>PR Check triggers</summary>
  
- Build: `trigger build`
- Sub Category Field Check: `trigger subcategoryfieldcheck`

### CODE REVIEW

**Review by RajBaviskar — APPROVED** [review]

File: internal/acctest/acctest.go [hunk 1]
```diff
@@ -120,6 +120,15 @@ func ProjectResourceImportStateIdFunc(resourceName string) resource.ImportStateI
...
 }
 
+func UserResourceImportStateIdFunc(resourceName string) resource.ImportStateIdFunc {
+	return func(s *terraform.State) (string, error) {
+		primary := s.RootModule().Resources[resourceName].Primary
+		orgId := primary.Attributes["org_id"]
+		projId := primary.Attributes["project_id"]
+		email := primary.Attributes["email"]
+		id := primary.ID
+		return fmt.Sprintf("%s/%s/%s/%s", orgId, projId, email, id), nil
```
Comment (rathodmeetsatish) [inline internal/acctest/acctest.go:130]: when we are asking for email we wont need to ask for id

**Review by rathodmeetsatish — CHANGES_REQUESTED** [review]
Please add examples

**Review by rathodmeetsatish — CHANGES_REQUESTED** [review]
add examples

**Comment by rijajoo** [comment]
trigger subcategoryfieldcheck

**Comment by rathodmeetsatish** [comment]
trigger build

**Comment by rathodmeetsatish** [comment]
trigger subcategoryfieldcheck

**Comment by rathodmeetsatish** [comment]
trigger subcategoryfieldcheck

**Comment by rijajoo** [comment]
trigger build

**Pull request merged**